### PR TITLE
[scripts] Implement generalized build_cxx_project.sh; configure SC_BIN_PATH from scripts

### PR DESCRIPTION
**Pull request by NikitaZotov**
* [x] Read PR [documentation](https://github.com/ostis-ai/sc-machine/blob/main/CONTRIBUTING.md)
* [x] Update changelog
* [x] Update documentation

### CODE REVIEW

**Review by MksmOrlov — CHANGES_REQUESTED** [review]
image

**Comment by MksmOrlov** [comment]
Install script doesn't install cmake?

File: scripts/set_vars.sh
Comment (MakarenkoAI) [inline scripts/set_vars.sh:1]: change access right. script isnt executable

File: scripts/build-scripts/build_cxx_project.sh
Comment (MakarenkoAI) [inline scripts/build-scripts/build_cxx_project.sh:1]: image
idk, maybe i must have run this from metasystem firstly or smth like that

File: scripts/set_vars.sh
Comment (NikitaZotov) [reply scripts/set_vars.sh:1]: It is not for manual run

File: scripts/build-scripts/build_cxx_project.sh
Comment (NikitaZotov) [reply scripts/build-scripts/build_cxx_project.sh:1]: Fixed

**Comment by NikitaZotov** [comment]
> Install script doesn't install cmake?

You ran not last version of scripts

**Review by MksmOrlov — APPROVED** [review]

**Review by MakarenkoAI — APPROVED** [review]

**Pull request merged**